pkgs/tools/security/doas/default.nix

```
@@ -3,6 +3,7 @@
 , fetchFromGitHub
 , bison
 , pam
+, enablePAM ? true
 
```

Keeping with the established style of the file:

```suggestion

, withPAM ? true
```